### PR TITLE
System directory to support experiments

### PR DESCRIPTION
**Pull request by tab-cmd**
# Overview

Added a system directory that contains an experiment and field registry. Updated save method to include an experiment. 

## Ticket

https://www.pivotaltracker.com/story/show/175193520

## Contributions

- Added `.bcipy`
- Added `.bcipy/experiment/`
- Added `.bcipy/field/`
- Added `.bcipy/experiment/experiments.json`
- Added `.bcipy/field/fields.json`
- Updated `helpers/save/init_save_data_structure` to include the experiment in the save structure. Defined a DEFAULT_EXPERIMENT_ID.
- Removed copying of the helpers folder (not useful to clinical team)
- Updated `bci_main` to include the setting of the experiment id.
- Updated helper save tests

## Test

- Unit tests

## Documentaion

- No

### CODE REVIEW

File: bcipy/helpers/save.py [hunk 1]
```diff
@@ -45,28 +49,13 @@ def init_save_data_structure(data_save_path,
...
-        for file_name in src_files:
-
-            # get the full name
-            full_file_name = os.path.join('bcipy/helpers/', file_name)
-
-            # Check that constructed file is a real file and ends in .py
-            if (os.path.isfile(full_file_name)) and '.py' in file_name:
-                # Copy over!
-                copy2(full_file_name, helper_folder_name)
-
```
Comment (lawhead) [inline bcipy/helpers/save.py:65]: When I was looking into BIDS I meant to ask if these helpers are still needed. I'm glad we're able to clean up the data.

**Review by lawhead — APPROVED** [review]

File: bci_main.py [hunk 2]
```diff
@@ -36,7 +38,12 @@ def bci_main(parameters: dict, user: str, exp_type: int, mode: str) -> bool:
 
     # Initialize Save Folder
     save_folder = init_save_data_structure(
-        data_save_location, user, parameter_location, mode, exp_type)
+        data_save_location,
+        user,
+        parameter_location,
+        mode=mode,
+        experiment_type=exp_type,
+        experiment=experiment)
```
Comment (lawhead) [inline bci_main.py:47]: bci_main is passing an extra parameter to init_save_data_structure which causes an error (`experiment`). Sorry I didn't catch this during the review.

Comment (lawhead) [reply bci_main.py:47]: Or actually, it looks like maybe this needs to be `experiment_id`, rather than `experiment`?

Comment (tab-cmd) [reply bci_main.py:47]: yeah - it should be experiment_id. I'll re-open a PR

**Pull request merged**